### ISSUE TITLE
Release Twisted 15.2

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#7844]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)| @hawkowl reported|
|-|-|
|Trac ID|trac#7844|
|Type|enhancement|
|Created|2015-04-13 08:33:18Z|
|Branch|https://github.com/twisted/twisted/tree/releases/release-15.2.0-7844|

Let's keep the pace up.

<details><summary>Searchable metadata</summary>

```
trac-id__7844 7844
type__enhancement enhancement
reporter__hawkowl hawkowl
priority__normal normal
milestone__None None
branch__branches_releases_release_15_2_0_7844 branches/releases/release-15.2.0-7844
branch_author__ 
status__closed closed
resolution__fixed fixed
component__release_management release management
keywords__ 
time__1428913998995592 1428913998995592
changetime__1445217226370668 1445217226370668
version__None None
owner__hawkowl hawkowl
cc__radix
```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_3"></a>|@hawkowl removed owner|
|-|-|

Builders spun, please review.

**Comment by twisted-trac** [comment]
|[<img alt="adiroiban's avatar" src="https://avatars.githubusercontent.com/u/204609?s=50" width="50" height="50">](https://github.com/adiroiban)<a name="note_4"></a>|@adiroiban commented|
|-|-|

Changes looks good. Many thanks for the release!

There is one problem in `Improved Documentation` section for [#7875](https://github.com/twisted/twisted/issues/7875) as it has no text.

I was the one merging it, so my bad.

Can you please move it into `Other` section or update it to something like: `Fix a typo in narrative documentation for logger`.

Thanks!

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)<a name="note_5"></a>|@hynek commented|
|-|-|

LGTM, go for it.

**Comment by twisted-trac** [comment]
|[<img alt="hynek's avatar" src="https://avatars.githubusercontent.com/u/41240?s=50" width="50" height="50">](https://github.com/hynek)|@hynek set owner to @hawkowl|
|-|-|

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_7"></a>|@hawkowl set status to `closed`|
|-|-|

(In [44861]) Merge releases/release-15.2.0-7844: Release Twisted 15.2

Author: hawkowl
Reviewers: adiroiban, hynek
Fixes: [#7844](https://github.com/twisted/twisted/issues/7844)

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_8"></a>|@hawkowl commented|
|-|-|

Ticket retargeted after milestone deleted
